src/FlexibleNumberKernel.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Kernel interface for base-flexible numeric representation.
+ */
+public interface FlexibleNumberKernel {
```

Comment:
Make sure this extends the kernel:
```suggestion
public interface FlexibleNumberKernel extends Standard<FlexibleNumber> {
```